src/Components/People.js

```diff
@@ -0,0 +1,104 @@
+import axios from "axios";
+import { useState } from "react";
+
+const People = () => {
+	// declare the state
+	const [userInput, setUserInput] = useState("");
+	const [person, setPerson] = useState([]);
+	const [personAge, setPersonAge] = useState("");
```

Comment:
It's better to have one piece of state that keeps track of the entire person and then you can access the age and other information from that piece of state